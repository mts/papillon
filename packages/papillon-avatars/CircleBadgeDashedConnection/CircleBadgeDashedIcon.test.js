import React from 'react';
import {
  circleBadgeDashedConnection,
} from './CircleBadgeDashedIcon.render';

describe('<Avatar />', () => {
  let wrapper;

  describe('Instance', () => {
    test('must be an instance of CircleBadgeDashedConnection', () => {
      wrapper = renderMount(circleBadgeDashedConnection);
      expect(wrapper.find('CircleBadgeDashedConnection')).toHaveLength(1);
    });
  });

  describe('Snaphot', () => {
    test('must match circleBadgeDashedConnection', () => {
      expect(global.renderToJSON(circleBadgeDashedConnection)).toMatchSnapshot();
    });
  });
});
