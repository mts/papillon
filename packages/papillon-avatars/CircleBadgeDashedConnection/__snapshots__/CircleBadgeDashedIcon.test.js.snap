// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> Snaphot must match circleBadgeDashedConnection 1`] = `
<div
  className="DashedConnection"
>
  <ul
    aria-label="A sample GitHub workflow"
    className="d-flex list-style-none flex-justify-between"
  >
    <li
      aria-label="GitHub"
      className="CircleBadge CircleBadge--small"
    >
      <img
        alt=""
        className="CircleBadge-icon"
        src="https://user-images.githubusercontent.com/334891/30004619-adc8ca86-90a0-11e7-8ccc-2322261d2ab1.png"
      />
    </li>
    <li
      aria-label="GitHub"
      className="CircleBadge CircleBadge--small"
    >
      <img
        alt=""
        className="CircleBadge-icon"
        src="https://user-images.githubusercontent.com/334891/30004619-adc8ca86-90a0-11e7-8ccc-2322261d2ab1.png"
      />
    </li>
    <li
      aria-label="GitHub"
      className="CircleBadge CircleBadge--small"
    >
      <img
        alt=""
        className="CircleBadge-icon"
        src="https://user-images.githubusercontent.com/334891/30004619-adc8ca86-90a0-11e7-8ccc-2322261d2ab1.png"
      />
    </li>
  </ul>
</div>
`;
